gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/DagActionStore.java

```diff
@@ -45,6 +44,11 @@ public DagAction(String flowGroup, String flowName, String flowExecutionId, DagA
       this.flowExecutionId = flowExecutionId;
       this.dagActionValue = dagActionValue;
     }
```

Comment:
won't `@Data` synthesize a ctor for us?  also, how does the synthesized `toString()` compare to the hand-coded one below?